src/lib/autocomplete/autocomplete.ts

```diff
@@ -33,7 +33,12 @@ let _uniqueAutocompleteIdCounter = 0;
 
 /** Event object that is emitted when an autocomplete option is selected */
 export class MatAutocompleteSelectedEvent {
-  constructor(public source: MatAutocomplete, public option: MatOption) { }
+  constructor(
+    /** Reference to the autocomplete panel that emitted the event. */
+    public source: MatAutocomplete,
+
```

Comment:
nit: for consistency, can you do the blank line in all of the files or none of them